### DISCUSSION TITLE
[Feature]: Automatically show new items on the website when uploading and when a new file has been uploaded

### DISCUSSION THREAD

**Discussion by JVT038** [immich-app/immich#1664] [Feature Request]
### Feature detail

It would be nice if I were browsing the website and I would see a notification that says new files have been added to the interface and that I can automatically see those items popping up in the timeline.
Also, when I add upload a new item on the website, it would be nice if it would automatically pop up, instead of having to refresh the page.

### Platform

Web

**Comment by alextran1502** [comment]
There was this feature before, since we implemented the virtual scroll, we temporarily disable this feature 😛. Will update this issue once it is reimplemented

**Comment by bo0tzz** [comment]
This is now supported
